internal/app/utils.go

```diff
@@ -57,7 +57,7 @@ func fromTOML(file string, s *state) (bool, string) {
 	if _, err := toml.Decode(tomlFile, s); err != nil {
 		return false, err.Error()
 	}
-	addDefaultHelmRepos(s)
+	//addDefaultHelmRepos(s)
```

Comment:
nit: if the function was removed there's no point in keeping this commented out here.